test/gtest/uct/test_pending.cc

```diff
@@ -107,6 +107,14 @@ class test_uct_pending : public uct_test {
         return (ucs_status_t)packed_len;
     }
 
+    static ucs_status_t pending_send_op_ok(uct_pending_req_t *self) {
+        pending_send_request_t *req = ucs_container_of(self, pending_send_request_t, uct);
+
+        delete req;
```

Comment:
it is better practice to have pair of routines alloc/free instead of direct call ```delete```